dill/_objects.py

```diff
@@ -111,6 +109,7 @@ class _Struct(ctypes.Structure):
         pass
     _Struct._fields_ = [("_field", ctypes.c_int),("next", ctypes.POINTER(_Struct))]
 _filedescrip, _tempfile = tempfile.mkstemp('r') # deleted in cleanup
+os.close(_filedescrip)
```

Comment:
That's because that line is not tested.  To check it, comment the former and uncomment the latter.
```
load_types(pickleable=True,unpickleable=False)
#load_types(pickleable=True,unpickleable=True)
```